resources/lang/front/nl/form.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+return [
+
+    'validation_error.title' => 'Je invoer is invalide.',
```

Comment:
Probably use 'uw' instead of 'je'? Also to keep it consistent with the next line after this one